support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -88,6 +88,27 @@ export const tests: Tests = {
     targetPage: pageUrlRegexes.contributions.auLandingPage,
     seed: 1,
   },
+  freeTrialLengthTest: {
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'variant',
+      },
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
```

Comment:
We should probably consider any other ab tests we intended to run on the Digi Subs landing page in the near future, such as the upcoming Comparison table AB test. We might want to set the `size` to `0.5` so the audience under test doesn't over lap?